codalab/rest/bundles.py

```diff
@@ -395,12 +396,16 @@ def _fetch_bundle_contents_info(uuid, path=''):
     ```
     """
     depth = query_get_type(int, 'depth', default=0)
+    target = BundleTarget(uuid, path)
     if depth < 0:
         abort(http.client.BAD_REQUEST, "Depth must be at least 0")
 
     check_bundles_have_read_permission(local.model, request.user, [uuid])
     try:
-        info = local.download_manager.get_target_info(uuid, path, depth)
+        info = local.download_manager.get_target_info(target, depth)
+        # Object is not JSON serializable so submit its dict in API response
+        # The client is responsible for deserializing it
+        info['resolved_target'] = info['resolved_target'].__dict__
```

Comment:
Here's another place where we send this over the API and need to serialize to dict.